### PR TITLE
Improve the sign apk script

### PR DESCRIPTION
**Pull request by ClaireGizard**
#371

### CODE REVIEW

**Review by Florian14 — COMMENTED** [review]
SGTM, can you add the usage in the script to handle cases like wrong parameters or the absence of apks in the given folder?

File: tools/release/sign_all_apks.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+if [[ $PARAM_KEYSTORE_PATH == ""  && $PARAM_DIRECTORY == "" ]]
+then
+  echo "Usage: ./sign_all_apks \$KEY \$FOLDER"
+  exit 1
+fi
+if [[ $PARAM_KEYSTORE_PATH == "" || $PARAM_DIRECTORY == "" ]]
+then
+  echo "Usage: ./sign_all_apks \$KEY \$FOLDER"
+  exit 1
+fi
```
Comment (Florian14) [inline tools/release/sign_all_apks.sh:14]: I am not sure both checks are necessary, also I suggest verifying the number of parameters using `"$#" -ne 1` in your condition

File: tools/release/sign_all_apks.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Get the command line parameters
+PARAM_KEYSTORE_PATH=$1
+PARAM_DIRECTORY=$2
+CHECKSUM_FILE="checksum.txt"
```
Comment (Florian14) [inline tools/release/sign_all_apks.sh:8]: you can put this bloc after the check proposed in my following comment

**Review by giomfo — APPROVED** [review]
LGTM

**Pull request merged**